EmVer/include/EmVerUtility.hpp

```diff
@@ -0,0 +1,5 @@
+#include "mbed.h"
+
+bool assertPin(PinName pinName, bool expected, int timeout);
+bool assertPinToggle(PinName pinName, int expectedOccurences, int timeout);
```

Comment:
I think for ease of use and cleaner test code I would suggest putting some defaults within here. Maybe pick a reasonable timeout (depends on the testcode but maybe 20 ms or so).  For expectedOccurences it would almost always be 2 as that is generally what a toggle is.